### ISSUE TITLE
sklearn integration to be done

### ISSUE THREAD

**Issue by prodicus** [prodicus/spamfilter#5]
Taking advantage of the various ML algorithms implemented in [`sklearn`](http://scikit-learn.org/stable/index.html) would help us make a better prediction. 

Will have to read upon the various implementations upon which we will chose our course of action

Ping here for ideas related to that
